docs/proposal/1_1_business_background.tex

```diff
@@ -1,5 +1,3 @@
-As the project is a proof-of-concept research project, the business aspects 
-typical to real-world engineering projects should not be applicable.
+The client of this project is Professor Mieszko Lis, a professor in the Electrical and Computer Engineering department at the University of British Columbia (UBC). A particular research focus of Dr. Lis is the acceleration of (traditionally software-based) machine-learning (ML) models through the use of dedicated hardware. 
 
-The suitor of this project is a team of undergraduate students at the University of British Columbia (UBC). The name of the team is Skynet. The members of Skynet are all students in the Faculty 
-of Applied Science in discipline of Electrical and Computer Engineering. The members of Skynet are all students taking the fourth year Engineering capstone course (ELEC491/CPEN491) at UBC, in which they have been assigned this engineering project. Relating to the context of the project, the members of Skynet all have experience working with FPGAs, and in RTL design.
\ No newline at end of file
+The fabrication of custom-designed hardware is often prohibitively expensive - the manufacture of a single application-specific integrated controller (ASIC) can easily cost millions of dollars and take several years to complete. This is inherently unconducive to the quick and iterative prototyping required by small-scale research projects like those run by Dr. Lis. To mitigate this constraint, Dr. Lis would like to use Field Programmable Gate Arrays (FPGAs) - effectively programmable analogs of a real circuit - to prototype his ML hardware designs. Actual hardware designs can be (rapidly) modeled on an FPGA, which acts exactly like a traditional circuit, allowing for rapid iterative prototyping. Once a finalized design has emerged from the prototyping process, it can then be manufactured into an actual hardware chip, if desired.
```

Comment:
nit: The ` - ` should be instead an _em dash_ to convey continuation of idea. Which is three hyphens: ---.

Reply:
Haha, let's meet in the middle and use an en dash instead